src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx

```diff
@@ -362,4 +362,8 @@ The first part date {1} should be lesser than or equal to the second part date {
     <value>The dicom file in the request section exceeded the allowed limit of {0} bytes.</value>
     <comment>{0} Max number of bytes allowed per dicom file.</comment>
   </data>
+  <data name="MismatchVR" xml:space="preserve">
+    <value>The VR code '{0}' for tag '{1}' does not match to expected VR '{2}'.</value>
```

Comment:
>The VR code '{0}' for tag '{1}' does not match to expected VR '{2} [](start = 11, length = 66)

Can you add more details here? Why was the match expected for example.

Reply:
Hi Sneah, I'm not sure what you mean here.  We  usually just put a brief error message here instead of fully details (not very sure if this is true, but seems that's a pattern for existing error messages).

Reply:
I think the additional details will help clarify what a match means! So something along the lines of 'Query tag '' is expected to have VR '' but has VR '' in file'.

---
In reply to: [619518813](https://github.com/microsoft/dicom-server/pull/736#discussion_r619518813) [](ancestors = 619518813)